flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/api/tableImpl.scala

```diff
@@ -502,7 +455,8 @@ class GroupedTableImpl(
 
   private def selectInternal(fields: Seq[Expression]): Table = {
     val expressionsWithResolvedCalls = fields.map(_.accept(tableImpl.callResolver)).asJava
-    val extracted = extractAggregationsAndProperties(expressionsWithResolvedCalls, tableImpl.getUniqueAttributeSupplier)
+    val extracted = extractAggregationsAndProperties(expressionsWithResolvedCalls,
```

Comment:
nit: one line per argument whenever it does not fit into a line makes the code more readable